aten/src/ATen/function_wrapper.py

```diff
@@ -1144,7 +1180,12 @@ def gen_tensor_method(option):
                 static_dispatch_method_body = STATIC_DISPATCH_FUNCTION_DEFAULT_BODY.substitute(
                     option, native_arguments=option['method_actuals'])
 
-            method_definition = (C10_TENSOR_METHOD_DEFINITION if option['use_c10_dispatcher'] else TENSOR_METHOD_DEFINITION)
+            if option['use_c10_dispatcher'] == 'no':
+                method_definition = TENSOR_METHOD_DEFINITION
+            elif option['use_c10_dispatcher'] == 'unboxed_only':
+                method_definition = C10_UNBOXEDONLY_TENSOR_METHOD_DEFINITION
+            else:
```

Comment:
Can we add an `assert['use_c10_dispatcher'] == 'full'` here? (or whatever that value is)

Reply:
There is an assertion in `native_parse.yaml` but I'll also add one here.